### PR TITLE
Upgrade all packages with pur

### PR DESCRIPTION
**Pull request by humitos**
```
$ for reqs in `ls requirements`; \
    do pur --skip redis,commonmark,django,docker,celery,gitpython,elasticsearch,pyelasticsearch,mercurial --requirement requirements/$reqs; \
done
```

### CODE REVIEW

**Comment by stsewd** [comment]
We could upgrade Django to `1.11.17` here too https://github.com/rtfd/readthedocs.org/issues/4756#issuecomment-450887566

**Comment by humitos** [comment]
Done!

File: requirements/lint.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
 -r pip.txt
 # We need these special cases of Python < 3 because we run tests with
 # Python 2 still
-astroid==2.0.4; python_version > '3'
-astroid==1.6.4; python_version < '3'
-pylint==2.1.1; python_version > '3'
+astroid==2.1.0; python_version > '3'
+astroid<1.7 python_version < '3'
```
Comment (stsewd) [inline requirements/lint.txt]: A semicolon is missing here

**Comment by agjohnson** [comment]
Perhaps you could add the command you run as an invoke task in our `tasks.py`? I think the only place it's documented right now is in our PRs upgrading

**Comment by humitos** [comment]
> Perhaps you could add the command you run as an invoke task in our `tasks.py`?

Done at https://github.com/rtfd/common/pull/27

**Pull request merged**